src/db.rs

```diff
@@ -307,5 +307,14 @@ CREATE UNIQUE INDEX jobs_name_scheduled_at_unique_index
     ON jobs (
         name, scheduled_at
     );
+",
+    "
+CREATE table review_prefs (
+    id UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+    user_id BIGINT REFERENCES users(user_id),
+    assigned_prs INT[] NOT NULL DEFAULT array[]::INT[],
+    num_assigned_prs INTEGER
```

Comment:
Why is `num_assigned_prs` stored in the DB? Isn't that just the length of `assigned_prs`? If yes, then it should be calculated using derived SQL queries, like `SELECT array_length(schedule, 1)`.

Reply:
I usually do that so later I can add an index and have good performances on queries that involve that field (f.e. *who has most PRs assigned?*).

But that table will have ... 200 records at most? 400? So probably the perf. gain is negligible? :man_shrugging:

Reply:
Yeah, that sounds like a big overkill :laughing: And breaks the DB normal form unnecessary, IMO. The array also seems fishy, since we don't really need the potential perf. gains here, but that at least reduces the number of requires tables.

Reply:
Removed the field in https://github.com/rust-lang/triagebot/pull/1754/commits/f8ff17d29f55e69b1fe2dbebe6febd76a12cf3fa